src/telemetry/opentelemetry.ts

```diff
@@ -77,13 +77,13 @@ const instrumentations = [
     mergeItems: true,
     ignoreTrivialResolveSpans: true,
   }),
-  new PinoInstrumentation({
-    logKeys: {
-      traceId: 'trace',
-      spanId: 'spanId',
-      traceFlags: 'traceSampled',
-    },
-  }),
+  // new PinoInstrumentation({
+  //   logKeys: {
+  //     traceId: 'trace',
+  //     spanId: 'spanId',
+  //     traceFlags: 'traceSampled',
+  //   },
+  // }),
```

Comment:
just remove the lines, we have it in history, also maybe add a backlog task for it?

Reply:
Lines removed

Reply:
Will create backlog task for it